### PR TITLE
test: Update visualization v2 to output HTML and Markdown. Partial #3970

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**
Partial #3970

outputhtml
outputmd

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: samples/test/metrics_visualization_v2.py [hunk 1]
```diff
@@ -113,9 +115,27 @@ def iris_sgdclassifier(test_samples_fraction: float, metrics: Output[Classificat
         confusion_matrix(train_y, predictions).tolist() # .tolist() to convert np array to list.
     )
 
+@component(
+    kfp_package_path='git+https://github.com/kubeflow/pipelines#egg=kfp&subdirectory=sdk/python'
```
Comment (chensun) [inline samples/test/metrics_visualization_v2.py:119]: Sorry, I was suggesting this for local testing. 
For tests, we should still use that environment variable which will install KFP under a commit hash.

Comment (zijianjoy) [reply samples/test/metrics_visualization_v2.py]: Sounds good, fixed.

File: samples/test/metrics_visualization_v2_test.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def verify(run: kfp_server_api.ApiRun, mlmd_connection_config, **kwargs):
     task_names = [*tasks.keys()]
     t.assertCountEqual(
         task_names,
-        ['wine-classification', 'iris-sgdclassifier', 'digit-classification'],
+        ['wine-classification', 'iris-sgdclassifier', 'digit-classification', 'html-visualization', 'markdown-visualization'],
```
Comment (Bobgy) [inline samples/test/metrics_visualization_v2_test.py:36]: Also verify the output artifact type? That seems crucial too

**Review by Bobgy — COMMENTED** [review]
/lgtm

File: samples/test/metrics_visualization_v2_test.py (hunk 2 above)
Comment (zijianjoy) [reply samples/test/metrics_visualization_v2_test.py:36]: It makes sense, added test validation.

**Comment by Bobgy** [comment]
FYI, I'm fixing e2e tests in https://github.com/kubeflow/pipelines/pull/6351

**Pull request merged**